Doc/library/sqlite3.rst

```diff
@@ -1255,17 +1254,22 @@ Cursor objects
          >>> cur.connection == con
          True
 
+.. The sqlite3.Row example used to be a how-to. It has now been incorporated
+   into the Row reference. We keep the anchor here in order not to break
+   existing links.
+
+.. _sqlite3-columns-by-name:
+
 .. _sqlite3-row-objects:
```

Comment:
```suggestion
.. _sqlite3-columns-by-name:
.. _sqlite3-row-objects:
```

Trivial, but any reason to leave a space here? I've usually seen multiple ref target labels without one, IIRC.